acceptance/bundle/templates/telemetry/test.toml

```diff
@@ -1,10 +1,6 @@
 RecordRequests = true
 IncludeRequestHeaders = ["User-Agent"]
 
-[[Repls]]
-Old = '17\d{11}'
-New = '"UNIX_TIME_MILLIS"'
-
```

Comment:
This was quoted to keep the request log valid JSON.

I recall discussing this with @shreyas-goenka a while back.